### PR TITLE
[Korean] Update vue-router 2.5

### PR DESCRIPTION
**Pull request by ChangJoo-Park**

### CODE REVIEW

File: docs/kr/advanced/navigation-guards.md [hunk 1]
```diff
@@ -32,8 +32,20 @@ router.beforeEach((to, from, next) => {
...
+  - **`next(error)`**: (2.4.0 이후 추가) `next`에 전달된 인자가 `Error` 의 인스턴스이면 탐색이 중단되고 `router.onError()`를 이용해 등록 된 콜백에 에러가 전달됩니다.
+
+
 **항상 `next` 함수를 호출하십시오. 그렇지 않으면 훅이 절대 불러지지 않습니다.**
 
+### Global Resolve Guards
+
+> 2.5.0에서 추가됨
+
+In 2.5.0+ you can register a global guard with `router.onResolve`. This is similar to `router.beforeEach`, with the difference that resolve guards will be called right before the navigation is confirmed, **after all in-component guards and async route components are resolved**.
```
Comment (kazupon) [inline docs/kr/advanced/navigation-guards.md]: Did you forget to remove it?

**Review by kazupon — CHANGES_REQUESTED** [review]
Thank you for your PR!
Please check my review comment 💬

File: docs/kr/advanced/navigation-guards.md (hunk 1 above)
Comment (ChangJoo-Park) [reply docs/kr/advanced/navigation-guards.md]: Oh, It is my mistake 😭

**Comment by ChangJoo-Park** [comment]
Sorry @kazupon :sob: I forgot to remove original sentences

**Review by kazupon — APPROVED** [review]

**Comment by kazupon** [comment]
Thanks!

**Comment by ChangJoo-Park** [comment]
:)

**Pull request merged**